compass/landice/tests/ismip6_run_ais/projection/__init__.py

```diff
@@ -0,0 +1,48 @@
+from compass.testcase import TestCase
+from compass.landice.tests.ismip6_run_ais.set_up_experiment \
+        import SetUpExperiment
+from compass.validate import compare_variables
+
+
+class Projection(TestCase):
+    """
+    A test case for performing forward MALI runs of ISMIP6 Antaractic setup
+    """
+
+    def __init__(self, test_group):
+        """
+        Create the test case
+
+        Parameters
+        ----------
+        test_group : compass.landice.tests.ismip6_run_ais.Ismip6RunAIS
+            The test group that this test case belongs to
+
+        """
+        name = 'ismip6AE'
+
+        super().__init__(test_group=test_group, name=name,
+                         subdir=name)
+
+        experiments = ['hist', 'ctrlAE', 'expAE01', 'expAE02', 'expAE03',
+                       'expAE04', 'expAE05', 'expAE06']
+        for i in range(7, 10+1):
+        #for i in range(7, 14+1):
+            experiments.append(f'expAE{i:02}')
+
+        for exp in experiments:
```

Comment:
It seems like it might make more sense to have one test case per experiment rather than one step per experiment.  Compass is designed to make it pretty easy to set up several test cases as a "custom" suite and then run them in the order you chose but it's a little trickier to go in and edit the `steps_to_run`  config option to run only the subset of steps you want to run in a given test case.

We can work with it either way, it's just what I would think might be easier.

Reply:
There are pros and cons to each layout, but after discussion with you and with Holly, my preference is to keep each experiment as a step.  One advantage to that is not have a bunch of experiments each with a single step in a subdirectory, but instead have each a test case with all the experiments as subdirectories.  I think after making the experiments to set up configurable in the cfg file, that provides the desired user flexibility.